### PR TITLE
Fix flash message causes content shift

### LINKED ISSUE #5103: "Thank you very much for improving that record" message causes content on page to shift
"Thank you very much for improving that record" message causes content on page to shift on page. This is like Lucy pulling the football away just as Charlie Brown is going to kick it; just as one goes to click on something, the target moves.

### Steps to Reproduce

1. Go to ...any work or edition record.
2. Do ...save an edit.

* Actual: "Thank you very much for improving that record"  appears briefly and then disappears, causing the rest of the content on page to shift up.
* Expected: The message should either remain until the patron clears it or behave in a way that does not cause content to shift.

### Details

- **Logged in (Y/N)?** Y
- **Browser type/version?** Version 90.0.4430.85 (Official Build) (x86_64)
- **Operating system?** Mac OS Big Sur
- **Environment (prod/dev/local)?** prod

### Proposal & Constraints

The message should either remain until the patron clears it or behave in a way that does not cause content to shift.

### Stakeholders

@seabelis

### PR DESCRIPTION
**Pull request by Yashs911**
Closes #5103 

Add a new class and position it accordingly.
### Screenshot

https://user-images.githubusercontent.com/64412143/116820240-90084200-ab91-11eb-9850-fb75fb22e998.mp4

### Stakeholders

@seabelis

### CODE REVIEW

File: openlibrary/templates/site/body.html [hunk 1]
```diff
@@ -22,7 +22,14 @@
       $#print errors (hidden by default as styles are loaded via JS)
       <div class="flash-messages" style="display: none;">
       $for flash in get_flash_messages():
-        <div class="$flash.type"><span>$flash.message</span></div>
+        <div class="$flash.type">
+          <span
+          $if flash.message == "Thank you very much for improving that record!":
```
Comment (mekarpeles) [inline openlibrary/templates/site/body.html:27]: I don't think we want to hard code messages here

Comment (Yashs911) [reply openlibrary/templates/site/body.html]: PR Updated:
Now we are using `get_message("flash_book_updated")`

**Review by cdrini — CHANGES_REQUESTED** [review]
Search for `flash-messages` in the codebase; there is a JS file, automatic.js that auto-hides these. Removing that one line should fix the problem.

**Comment by Yashs911** [comment]
@cdrini But then patrons won't be able to remove the message. So should I add a close button?
https://github.com/internetarchive/openlibrary/blob/43f892748a2b0d85e52dccf78c0cc66ff0462f2a/openlibrary/plugins/openlibrary/js/automatic.js#L11-L12

**Comment by cdrini** [comment]
They can refresh the page if it's really annoying, and it'll disappear. I think it's a net win over the content shift :)

**Comment by Yashs911** [comment]
PR updated

**Comment by RayBB** [comment]
@cdrini this was updated to do what you requested. Excited to see issue resolved!

**Review by cdrini — APPROVED** [review]
Tested on testing.openlibrary.org that making an edit no longer causes a layout shift. There's a bit of shift because the banner appears unstyled for a moment? Considering that ok for now.

**Comment by cdrini** [comment]
CI seems to have failed for no good reason :/ Something about ubuntu.com returning the wrong response 😒

**Pull request merged**